backend/src/main/java/kr/momo/service/confirmschedule/ConfirmScheduleService.java

```diff
@@ -0,0 +1,69 @@
+package kr.momo.service.confirmschedule;
+
+import java.time.LocalDate;
+import kr.momo.domain.attendee.Attendee;
+import kr.momo.domain.attendee.AttendeeRepository;
+import kr.momo.domain.availabledate.AvailableDate;
+import kr.momo.domain.availabledate.AvailableDateRepository;
+import kr.momo.domain.availabledate.AvailableDates;
+import kr.momo.domain.confirmedschedule.ConfirmedSchedule;
+import kr.momo.domain.confirmedschedule.ConfirmedScheduleRepository;
+import kr.momo.domain.meeting.Meeting;
+import kr.momo.domain.meeting.MeetingRepository;
+import kr.momo.exception.MomoException;
+import kr.momo.exception.code.AttendeeErrorCode;
+import kr.momo.exception.code.ConfirmedScheduleErrorCode;
+import kr.momo.exception.code.MeetingErrorCode;
+import kr.momo.service.schedule.dto.ScheduleConfirmRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class ConfirmScheduleService {
+
+    private final MeetingRepository meetingRepository;
+    private final AttendeeRepository attendeeRepository;
+    private final AvailableDateRepository availableDateRepository;
+    private final ConfirmedScheduleRepository confirmedScheduleRepository;
+
+    @Transactional
+    public void confirmSchedule(String uuid, long attendeeId, ScheduleConfirmRequest request) {
+        Meeting meeting = meetingRepository.findByUuid(uuid)
+                .orElseThrow(() -> new MomoException(MeetingErrorCode.INVALID_UUID));
+
+        Attendee attendee = attendeeRepository.findByIdAndMeeting(attendeeId, meeting)
+                .orElseThrow(() -> new MomoException(AttendeeErrorCode.INVALID_ATTENDEE));
+
+        validateHostPermission(attendee);
+
+        confirmedScheduleRepository.findByMeeting(meeting).ifPresent(ignore -> {
+            throw new MomoException(ConfirmedScheduleErrorCode.CONFIRMED_SCHEDULE_EXISTS);
+        });
```

Comment:
confirmedScheduleRepository.findByMeeting(meeting)
```suggestion
        confirmedScheduleRepository.findByMeeting(meeting)
                .ifPresent(ignore -> new MomoException(ConfirmedScheduleErrorCode.CONFIRMED_SCHEDULE_EXISTS));
```
중괄호 빼면 좋을 것 같아요👍

Reply:
재즈의 리뷰를 반영하며 수정하였습니다